### PR TITLE
feature branch ready for code review

### PR DESCRIPTION
**Pull request by SofiBretz**
### Summary of Changes:

- Created model
- Created validations 
- Created forms using the form_for tag
- Readme updated
- Rubocop, no offenses detected.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ In this project we build forms, both using nearly-pure HTML and then graduating
 * Run the rails server
 * Start testing the forms
 
-## Authors - click on the link below to see the authors' github
+## Authors
```
Comment (samgaco) [inline README.md]: _Please, for the next project try to make the PR from the feature branch you were working on originally, that way is easier to review_

Comment (rmauritsson) [reply README.md]: We will, Thanks

**Comment by rmauritsson** [comment]
### Summary of Changes

- Changed routes to include :only

File: config/routes.rb [hunk 2]
```diff
@@ -1,3 +1,3 @@
 Rails.application.routes.draw do
-  resources :users
+  resources :users, only: [:new, :create, :update]
```
Comment (samgaco) [inline config/routes.rb]: :1st_place_medal:

**Pull request merged**